src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -27,3 +28,25 @@ export const fetchingOrganizations = () => dispatch =>
     .fetchOrganizations()
     .then(payload =>
       dispatch(fetchOrganizations(payload)));
+
+export const fetchDictionaries = (
```

Comment:
maybe we should get some member contribution on the same but i think this would look  much neater in the api.js file. 
Something like
 `export const fetchDictionaries = (query, limit, page) => dispatch =>
   api.dictionaries
     .fetchDictionaries(query, limit, page)`
then you can copy everything else here to there. (This is just but a suggestion just to keep consistency)

Reply:
Done